style.css

```diff
@@ -478,7 +475,147 @@ textarea:focus {
   font-size: 32px;
 }
 
+.pop-up {
+  display: none;
+}
+
+.pop-up-menu {
+  display: block;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: 10px;
+  background-color: #fff;
+  z-index: 10;
+  color: #3c3a39;
+  justify-content: center;
+  bottom: 0;
+  border: 1px solid #dfe1e6;
+}
+
+/* .multi-x {
+  display: flex;
+  flex-direction: row;
+  width: 259px;
+  height: 82px;
+  font-size: 32px;
+} */
+
+.pop-up-menu .pop-title {
+  font-family: "roboto", sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.msgs {
+  color: #344563;
+  font-size: 16px;
+  width: 311px;
+  height: 220px;
+  margin: 12px 0;
+  order: 0;
+  line-height: 24px;
+  font-weight: 400;
+  flex-grow: 0;
+}
+
+.multi-mid {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.puOut {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  padding: 0 10px;
+}
+
+.puOut li {
+  padding: 10px 10px;
+  background: rgba(255, 255, 255, 0.24);
+  margin-right: 10px;
+  border: 1px solid #dfe1e6;
+}
+
+.design {
+  display: none;
+}
+
+.pop-desk-title {
+  display: none;
+}
+
 @media (min-width: 990px) {
+  .pop-desk {
+    display: flex;
+  }
+
+  .desk-head {
+    display: none;
+  }
+
+  .pop-title {
+    display: none;
+  }
+
+  .pop-desk-title {
+    display: block;
+    font-family: "Create Round", sans-serif;
+    font-size: 32px;
+    font-weight: 400;
+    color: #172b4d;
+  }
+}
+
+@media (min-width: 990px) {
+  .pop-up-menu {
+    position: absolute;
+    display: block;
+    border-radius: 5px;
+    box-shadow: rgb(100 100 111 / 20%) 0 7px 29px 0;
+    padding: 35px;
+    margin: 35px;
+    height: max-content;
+  }
+
+  .puOut li {
+    font-family: "Poppins", sans-serif;
+    font-size: 15px;
+  }
+
+  .pop-img {
+    flex: 1;
+  }
+
+  .snap {
+    width: 100%;
+  }
+
+  .desk-msgs {
+    flex: 1;
+    padding: 5px 70px;
+  }
+
+  .msgs {
+    width: fit-content;
+    font-size: 30px;
+    margin-bottom: 74px;
+  }
```

Comment:
- [ ] Please try to match the figma design, currently the font and line height of the text are different, please check: 

Yours  | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/151183639-9de13631-4a2f-4b7a-a660-6e064a508716.png)  | ![image](https://user-images.githubusercontent.com/77038610/151183688-1c3b68df-3419-4b86-9bff-0c61bfe73ed7.png)

To make it the same you can check the properties provided in Figma on the right side menu: 

![image](https://user-images.githubusercontent.com/77038610/151183833-aedf97dc-a1db-42df-9d97-eae974037e69.png)
 
Font family should be `inter` and  `line-height `has to be `30px`